packages/e2e-tests/plugins/inner-blocks-renderappender/index.js

```diff
@@ -0,0 +1,104 @@
+( function() {
+	var wp = window.wp;
```

Comment:
Aside: Not introduced here, but these tests always run in Chrome, so we should feel comfortable comfortable to use ES2015+ features, even when not transpiled, since they're supported natively.